hepmc.sh

```diff
@@ -1,7 +1,7 @@
 package: HepMC
 version: "%(tag_basename)s"
-tag: v2.06.10
-source: https://gitlab.cern.ch/hepmc/HepMC
+tag: HEPMC_02_06_10
+source: https://gitlab.cern.ch/hepmc/HepMC.git
```

Comment:
The source had to be modified

from 
    https://gitlab.cern.ch/hepmc/HepMC
to 
    https://gitlab.cern.ch/hepmc/HepMC.git

Note the ".git" missing at the end ; that is what prevents the HepMC sources to be downloaded apparently...
I have figured it out by looking at the alidist recipes provided for 'private' sources distributed via gitlab that we have in alisw, like epos.sh and dmpjet.sh.
This seems to cure the issue : it compiles under my alidock build at least.

(trying to recomment to figure out some features of GitBub Pull Request, would like to re-trigger the checks...)

Reply:
It's enough that you do a commit or simply wait a bit. Anyways, it looks like it's working now, so I am merging.

Reply:
Hello Giulio,
  thank you for this. Good piece of news.

Cheers,
  Antonin